### PR TITLE
ci: Configure the GitHub Actions

### PR DESCRIPTION
**Pull request by taorepoara**
## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [ ] I included unit tests that cover my changes
- [ ] I added/updated the documentation about my changes
- [x] I made my own code-review before requesting one

## Description of the changes
Configure the GitHub Actions to build, test and release the project.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/task.md
Comment (jonas-martinez) [inline .github/ISSUE_TEMPLATE/task.md:21]:
```suggestion
- Can we minimize the data sent to the network ? 
```

**Review by jonas-martinez — APPROVED** [review]
Looks good to me !
Waiting for a review from @shiipou just to be sure.

File: .github/workflows/build_ci.yml
Comment (shiipou) [inline .github/workflows/build_ci.yml:204]:
```suggestion
```

File: .github/workflows/build_ci.yml
Comment (shiipou) [inline .github/workflows/build_ci.yml:210]: Maybe use it with only one line : 
```suggestion
      run: npx -p conventional-changelog-conventionalcommits@5 -p @semantic-release/git -p semantic-release-export-data -p semantic-release@19 semantic-release
```

File: .github/workflows/build_ci.yml
Comment (shiipou) [inline .github/workflows/build_ci.yml:34]:
```suggestion
```

File: .github/workflows/build_ci.yml
Comment (shiipou) [inline .github/workflows/build_ci.yml:40]: Idem 
```suggestion
      run: npx -p conventional-changelog-conventionalcommits@5 -p @semantic-release/git -p semantic-release-export-data -p semantic-release@19 semantic-release --dry-run
```

**Review by shiipou — CHANGES_REQUESTED** [review]

**Pull request merged**
